inputs/tcp/tcp.go

```diff
@@ -48,19 +48,25 @@ func (l *TcpInput) Run(output chan common.MapStr) error {
   logp.Info("[TcpInput] Running TCP Input")
   server, err := net.Listen("tcp", ":" + strconv.Itoa(l.Port))
   if server == nil {
-      panic("couldn't start listening: " + err.Error())
+      // question - should we really quit completely if we can't bind to a single socket?
+      // or just log an error and keep going
+      panic ("couldn't start listening: " + err.Error())
```

Comment:
We should log an error, although the quit completely thing is a common behavior for Elastic's other tools.